authentication.md

```diff
@@ -138,6 +138,12 @@ There are multiple ways of transmitting access tokens:
 * as an `application/x-www-form-urlencoded` parameter: `access_token=[access token]`
 * inside the request header: `Authorization: Bearer [access token]`
 
+## Authorization errors
+
+Most of the API nodes require authentication, and if the token wasn't provided or it was wrong the server will reply with status code `401 Unauthorized`.
+
+If the client tries to access the scope which was not approved at the token request time then the server will respond `403 Forbidden`.
```

Comment:
If the client tries to access an OAuth scope that was not part of the authorization, the API will return `403 Forbidden`.